docs/getting_started.rst

```diff
@@ -160,11 +164,13 @@ Install Node.js, Yarn and other dependencies
 
 The Python project-specific dependencies installed above will install ``nodeenv``, which is a useful tool for using specific versions of Node.js and other Node.js tools in Python environments. To setup Node.js and Yarn within the Kolibri project environment, ensure your Python virtual environment is active, then run:
 
+For a more detailed guide to using nodeenv see :doc:`/howtos/nodeenv`.
```

Comment:
This sentence should go above, before the ``To setup Node.js and Yarn within the Kolibri project environment...``